.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Kindly add the name of the test file here because it's in the project requirements